flatpak/org.prismlauncher.PrismLauncher.yml

```diff
@@ -27,6 +27,16 @@ finish-args:
   - --filesystem=xdg-download:ro
 
 modules:
+  - name: cmark
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMARK_TESTS=OFF
+      - -DCMARK_BUILD_TYPE=Release
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/commonmark/cmark.git
+    builddir: true
```

Comment:
instead of this, it might be sufficient to enable `builddir` for the main `prismlauncher` module

Reply:
Good idea, changed that too